meteor/sofie-server/model/entities/test/entity-mock-factory.ts

```diff
@@ -0,0 +1,209 @@
+import { Segment, SegmentInterface } from '../segment'
+import { Part, PartInterface } from '../part'
+import { Piece, PieceInterface } from '../piece'
+import { PieceType } from '../../enums/piece-type'
+import { PieceLifespan } from '../../enums/piece-lifespan'
+import { Rundown, RundownInterface } from '../rundown'
+import { anything, instance, mock, when } from 'ts-mockito'
+import { TransitionType } from '../../enums/transition-type'
+import { PartTimings } from '../../value-objects/part-timings'
+
+export class EntityMockFactory {
+	public static createRundown(rundownInterface?: Partial<RundownInterface>): Rundown {
+		const mockedRundown: Rundown = this.createRundownMockInstance(rundownInterface)
+		return instance(mockedRundown)
+	}
+
+	public static createRundownMockInstance(rundownInterface?: Partial<RundownInterface>): Rundown {
+		if (!rundownInterface) {
+			rundownInterface = {} as RundownInterface
+		}
```

Comment:
Depends on if you rather want to write `rundownInterface?.x` everytime you need to call it

Reply:
Personally I'd prefer `rundownInterface?.x`, since we are creating an empty object just to access attributes that doesn't exist to have a fallback value for that attribute.

Reply:
I will update that for my next PR :)